products/containeranalysis/terraform.yaml

```diff
@@ -21,6 +21,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "container_analysis_note_basic"
         primary_resource_id: "note"
+        min_version: beta
```

Comment:
Just curious, do you know why this + the other example's version had to be added, instead of being inferred like before?

Reply:
They shouldn't need to, but I think it makes sense to keep them consistent even if it's a little redundant.

Reply:
👍 thanks!